frontend/src/v5/store/tickets/tickets.helpers.ts

```diff
@@ -185,3 +183,5 @@ export const templateAlreadyFetched = (template: ITemplate) => {
 	const fetchedProperties = ['modules', 'properties', 'config'];
 	return fetchedProperties.some((prop) => Object.keys(template).includes(prop));
 };
+
+export const getPropertiesInCamelCase = (properties) => mapKeys(properties, (_, key) => camelCase(key));
```

Comment:
https://lodash.com/docs/4.17.15#mapKeys
You can't do this because the first argument taken by mapKeys is actually the value (and not the key). This value then gets passed on to camelCase